apps/src/aichat/openai.ts

```diff
@@ -0,0 +1,22 @@
+import {ChatCompletionMessage} from './types';
+import HttpClient from '@cdo/apps/util/HttpClient';
+
+export async function openaiCompletion(
```

Comment:
nit: could we make this function name a bit clearer? something like `get/fetchOpenaiCompletion` or `get/fetchChatCompletion`?

Reply:
That works!